specs/delete-file-share-spec.adoc

```diff
@@ -0,0 +1,110 @@
+= Delete file share workflow by using tendrl-frontend UI
+
+The intent of delete file share workflow is to provide a mechanism to delete a file share along with all connected clients and associated bricks.
+
+* Steps: 
+
+    - This action (file share delete) can be triggered in any of the 3 places: 
+
+        ** List file share - delete action.
+
+        ** Cluster object details -> File share tab - delete action.
+
+        ** File share object details - delete action
+
+    - Delete file share wizard will be modal popup, which will populate following information:
+
+    	** Warning message to draw attention to the fact that this file share will be permanently deleted.
+
+    	** And provide current utilization , a list of clients accessing this file share and a summary of bricks and usage.
```

Comment:
@anivargi @shtripat @nnDarshan Can i have some sample response data for file share's utilization and summary of bricks with usage for specific file share.